app/lib/usage_report.rb

```diff
@@ -7,8 +7,8 @@ def self.generate
     Jbuilder.encode do |report|
       report.id SiteSettings.anonymous_usage_id
       report.version do |version|
-        version.app ENV.fetch("APP_VERSION", "unknown").split(":")[-1]
-        version.sha ENV.fetch("GIT_SHA", "unknown")
+        version.app ENV.fetch("APP_VERSION", Rails.application.config.app_version).split(":")[-1]
```

Comment:
```suggestion
        version.app ENV.fetch("APP_VERSION", Rails.application.config.app_version)
```

If this split moves to application.rb as suggested, we don't need it here.